pkg/cmd/repo/rename/rename.go

```diff
@@ -144,6 +148,12 @@ func renameRun(opts *RenameOptions) error {
 		fmt.Fprintf(opts.IO.Out, "%s Updated the %q remote\n", cs.SuccessIcon(), remote.Name)
 	}
 
+	if opts.RenameLocalDir {
+		if err = renameRepoDir(newRepoName, opts); err != nil {
+			return err
+		}
+	}
+
```

Comment:
🤔 There's some awkwardness in putting this here in that the command will exit `1` if we can't rename the local directory, but the upstream repo name will have been changed. I think there's two approaches we can take to mitigate this:

1. Try to rename the directory before the repo, so the repo rename doesn't happen if the local directory rename happens. We'll probably want to change the name back to the original name, though, if the upstream repo name change fails... This could be awkward/clunky.
2. Print to `stderr` a message on local dir rename failure but still exit `0` (don't return the error).

I think I'm leaning towards the latter instead of the former, because the `gh repo rename` command will still have executed successfully even if the local directory rename fails. The change might look something like this:

```suggestion
	if opts.RenameLocalDir {
		if err = renameRepoDir(newRepoName, opts); err != nil {
			fmt.Fprintf(opts.IO.ErrOut, "Failed to rename local directory: %v", err)
		}
	}

```

Reply:
Yeah, the second option seems simpler and more reasonable than the first one